sass/variables/_colors.scss

```diff
@@ -183,3 +183,7 @@ $form-border-color-focus: $alert-primary;
 $form-background-color: $white;
 $form-input-color: $body-text-color;
 $form-placeholder-color: $grey-semi-dark;
+
+// Tables
+$table-row-divider-color: #b0b0b0;
```

Comment:
is this our 9th shade of grey? will you please add it to the greys on [lines 39-46](https://github.com/cb-talent-development/employer-style-base/blob/master/sass/variables/_colors.scss#L39). It can be `$grey-semi-darker` I guess